docs/topic/features.md

```diff
@@ -34,9 +34,23 @@ the data. This is very commonly used by organizations that provide big datasets
 on Google Cloud storage, to sustainably share costs of maintaining the data.
 
 When this feature is enabled, users on a hub accessing cloud buckets from
-other organizations marked as 'requestor pays' will increase our cloud bill.
+other organizations marked as 'Requester Pays' will increase our cloud bill.
 Hence, this is an opt-in feature.
 
+```{important}
+This feature enables the hub users to access `Requester Pays` buckets,
+**outside** of their project.
+
+However, note that this feature **does not** control which buckets **inside**
+the project will have `Requester Pays` enabled for themselves.
+
+This can be checked from the console following these steps in the
```

Comment:
Ahhh!! The last missing puzzle piece in my head. Thanks for pointing this out. I've added two commits trying to clarify this. There's also a technical example, because _I don't think_ we have done this before?

(I will also follow-up on the freshdesk ticket as well).